pages/Javascript-API.md

```diff
@@ -44,6 +44,39 @@ Each object contains sub-objects which correlate to each element's subelements a
 
 If changes are made to the `config` object, you must call `scene.updateConfig()` for them to take effect.
 
+####`scene.config.textures`
```

Comment:
Section looks OK but we should verify these and give examples. Also, this texture info doesn't necessarily belong in the JS API page since it's primarily about modifying the texture objects. Probably makes more sense in the texture section?